Dockerfile

```diff
@@ -1,11 +1,14 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 # Set Virtuoso commit SHA to Virtuoso 7.2.4 release (25/04/2016)
 ENV VIRTUOSO_COMMIT 96055f6a70a92c3098a7e786592f4d8ba8aae214
 
 # Build virtuoso from source and clean up afterwards
 RUN apt-get update \
+        && apt-get dist-upgrade -y \
```

Comment:
Docker best practices recommend avoiding dist-upgrade in dockerfile, see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/